core/src/main/scala/com/mesosphere/usi/core/japi/Scheduler.scala

```diff
@@ -1,12 +1,18 @@
 package com.mesosphere.usi.core.japi
 
+import java.util.concurrent.CompletableFuture
+
 import akka.NotUsed
-import akka.stream.javadsl
+import akka.stream.javadsl.{Flow, Sink, Source}
```

Comment:
Please use the `javadsl` prefix so that it is clear what is returned.
```suggestion
import akka.stream.javadsl
```